pytorch_lightning/core/lightning.py

```diff
@@ -54,7 +54,6 @@
 
 
 class LightningModule(
-    ABC,
```

Comment:
You'll need to remove the import too:

`pytorch_lightning/core/lightning.py:22: [F401] 'abc.ABC' imported but unused`

Reply:
it is my first time sir i am totally confuse please help me sir ....